lib/index.js

```diff
@@ -1,19 +1,48 @@
 // This file is in the entry point in your webpack config.
 import './styles.scss';
 import CurrentWeather from './current-weather'
+import BackgroundImage from './background-image'
 
-$('form').on('submit', function(event) {
+$(document).ready(function() {
+  $('#location-form').on('submit', getLocationInfo);
+  $('#sign-up').on('submit', registerUser);
+})
+
+const registerUser = (event) => {
```

Comment:
Related to item 1 in your PR, I'm not sure if you have worked with [classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) at all but creating a User class in a separate file would work well here and you can move your user related functions to be methods on that class like you said. All you need to do in your index.js file then is create a new instance of the class User: `const user = new User(email, password, confPassword)`. Happy to talk through how to structure that class more if you'd like.